src/theme/themeStyles.ts

```diff
@@ -99,6 +99,13 @@ const materialTheme = createTheme({
         },
       },
     },
+    MuiAutocomplete: {
+      styleOverrides: {
+        option: {
+          fontSize: '0.875rem',
```

Comment:
The sizes specified in rem (root em) units should not change for different browsers. The rem unit is relative to the root element's font size, which is typically set to a base value of 16 pixels (or another value defined in the CSS). As long as the root element's font size remains consistent across browsers, the rem units will yield the same results.

Different browsers may have slightly different default font sizes or zoom levels, which could indirectly affect the rendering of rem units. To ensure consistency, it's recommended to set a base font size explicitly.

Currently css in web-app
html {
  font-size: 100%
}

body {
   font-size: 0.875rem;
}

So there is no base font size set currently.

Reply:
I've created an issue #1809  to make font size and related properties consistent across the app.